File: ValidAnagram.java
import java.util.*;
class Solution {
    public boolean isAnagram(String s, String t) {
        char[] set1 = s.toCharArray();
        char[] set2 = t.toCharArray();

        Arrays.sort(set1);
        Arrays.sort(set2);

        return(Arrays.equals(set1, set2));
    }
}

File: ValidAnagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) != len(t):
            return False
        map1, map2 = {}, {}
        for i in range(len(s)):
            map1[s[i]] = map1.get(s[i], 0) + 1
            map2[t[i]] = map2.get(t[i], 0) + 1
        
        for c in map1:
            if map1[c] != map2.get(c, 0):
                return False
        return True
        